app/parse.py

```diff
@@ -15,8 +27,85 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELD = [field.name for field in fields(Product)]
+
+_driver: WebDriver | None = None
```

Comment:
Do you really need a global variable for WebDriver? For what? You can pass this variable as an argument to function.